app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -42,31 +54,164 @@ class CollectionTest {
         mockWebServer.shutdown()
     }
 
-    @Ignore
     @Test
-    fun AddTabToCollectionTest() {
-        // open a webpage, and add currently opened tab to existing collection
+    @Ignore("Temp disable test - see: https://github.com/mozilla-mobile/fenix/issues/5793")
+    // open a webpage, and add currently opened tab to existing collection
+    fun addTabToCollectionTest() {
+        val firstWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        val secondWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 2)
+
+        createCollection(firstWebPage.url, "testcollection_1")
+
+        // Close the open tab
+        homeScreen {
+            verifyHomeScreen()
+        }
+        Espresso.onView(ViewMatchers.withId(R.id.close_tab_button)).click()
+
+        // On homeview, open another webpage
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(secondWebPage.url) {
+            verifyPageContent(secondWebPage.content)
+        }
+
+        // Save the current page to the testcollection_1
+        navigationToolbar {
+        }.openThreeDotMenu {
+            // click save to collection menu item, type collection name
+            clickBrowserViewSaveCollectionButton()
+            org.mozilla.fenix.ui.robots.mDevice.wait(
+                Until.findObject(By.text("testcollection_1")),
+                TestAssetHelper.waitingTime)
+            Espresso.onView(ViewMatchers.withText("testcollection_1")).click()
+            Thread.sleep(5000)
```

Comment:
Having this sleep looks brittle, but I think you may have explained why this was necessary? If you must use a sleep here, perhaps could we leave a comment why it's necessary?  Also, instead of leaving time integer here, could we instead define that with the other time constants?  https://github.com/mozilla-mobile/fenix/blob/master/app/src/androidTest/java/org/mozilla/fenix/helpers/TestAssetHelper.kt#L18-L19   Note: Eventually, we should probably move all of them to the more generic TestHelper since they're being used for things other than test assets.

Reply:
Ah, this sleep was for investigation purpose, good catch.